### DISCUSSION TITLE
Inav 4.1 PIDs

### DISCUSSION THREAD

**Discussion by rhoads6257** [iNavFlight/inav#8052] [Q&A]
Built a 9" quad with 2212 kv1250 motors. Need some idea on where to start on PIDs. 4.1 only has 3 to 7" and my won't even get off the ground before wobbling and the flipping.

**Comment by DzikuVx** [comment]
Step 1 - veryfy filtering, probably have to lower gyro LPF to around 80-90Hz
Step 2 - push P gains on roll and pitch in steps of 10 until wit will stop shaking
